diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0cd32df..070be33 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -363,6 +363,11 @@ if(ENABLE_XPDF_HEADERS)
     goo/FixedPoint.h
     goo/gstrtod.h
     DESTINATION include/poppler/goo)
+  if(PNG_FOUND)
+    install(FILES
+      goo/PNGWriter.h
+      DESTINATION include/poppler/goo)
+  endif(PNG_FOUND)
   install(FILES
     fofi/FoFiBase.h
     fofi/FoFiEncodings.h
 
diff --git a/glib/CMakeLists.txt b/glib/CMakeLists.txt
index 6ed9523..9f5d0e7 100644
--- a/glib/CMakeLists.txt
+++ b/glib/CMakeLists.txt
@@ -30,6 +30,7 @@ set(poppler_glib_public_headers
   poppler-attachment.h
   poppler-form-field.h
   poppler-annot.h
+  poppler-layer.h
   poppler.h
 )
 
diff --git a/qt4/src/Makefile.am b/qt4/src/Makefile.am
index 7e982e7..10ac221 100644
--- a/qt4/src/Makefile.am
+++ b/qt4/src/Makefile.am
@@ -16,8 +16,7 @@ poppler_include_HEADERS =			\
 	poppler-form.h				\
 	poppler-optcontent.h			\
 	poppler-export.h			\
-	poppler-page-transition.h		\
-	poppler-page-transition-private.h
+	poppler-page-transition.h
 
 lib_LTLIBRARIES = libpoppler-qt4.la
 
