diff -ur Q7Z-orig/Source/Thread.py Q7Z/Source/Thread.py
--- Q7Z-orig/Source/Thread.py	2009-04-23 10:12:24.000000000 +0300
+++ Q7Z/Source/Thread.py	2009-05-29 03:46:02.541667405 +0300
@@ -220,7 +220,7 @@
 				sExt = ".exe"
 
 			# Recursive
-			slCmdArgs.append( "-r" )
+			#slCmdArgs.append( "-r" )
 
 			# Exclusions
 			slWildcards = QtCore.QStringList( Settings.User.Syntax.sExclusions.split( ' ' ) )
@@ -549,7 +549,7 @@
 			slCmdArgs.append( "-y" )
 
 			# Recursive
-			slCmdArgs.append( "-r" )
+			#slCmdArgs.append( "-r" )
 
 			# Exclusions
 			slWildcards = QtCore.QStringList( Settings.User.Syntax.sExclusions.split( ' ' ) )
