diff -ur Q7Z-orig/Source/Settings.py Q7Z/Source/Settings.py
--- Q7Z-orig/Source/Settings.py	2009-04-23 01:38:10.000000000 +0300
+++ Q7Z/Source/Settings.py	2009-05-30 00:42:50.391474549 +0300
@@ -39,7 +39,7 @@
 
 # Application
 Application				= QtCore.QObject()
-Application.sName		= "Q7Z"
+Application.sName		= "q7z"
 Application.sVersion	= QtCore.QString( "0.8.0" )
 Application.sDomain		= "k7z.sourceforge.net"
 Application.sCurrent	= "/7Z/" + Application.sName + "/Current.txt"
@@ -98,10 +98,10 @@
 Path				= QtCore.QObject()
 Path.sSep			= QtCore.QString( QtCore.QDir.separator() )
 Path.App				= QtCore.QObject()
-Path.App.sPrefix	= ""
+Path.App.sPrefix	= "/usr/share/" + Application.sName
 Path.App.sSBin		= ""
-Path.App.sInstall		= "/share/" + Application.sName
-Path.App.sConfig		= Env.sHome + '/.config/' + Author.sName + Path.sSep + Application.sName + Path.sSep
+Path.App.sInstall		= ""
+Path.App.sConfig		= Path.App.sPrefix
 Path.KDE			= QtCore.QObject()
 Path.KDE.sAutoStart	= ""
 Path.KDE.sConfig	= ""
@@ -270,20 +270,20 @@
 		# Misc
 		Path.KDE.sAutoStart = Path.KDE.sLocalPrefix + "/Autostart/"
 
-		# Installation
-		# KDE
-		if QtCore.QFile.exists( Path.Sys.sPrefix + Path.App.sInstall + "/Source/" ) :
-			Path.App.sPrefix	= Path.Sys.sPrefix
-			Path.App.sSBin		= Path.App.sPrefix + "/sbin"
-		# User
-		elif QtCore.QFile.exists( Path.Local.sPrefix + Path.App.sInstall + "/Source/" ) :
-			Path.App.sPrefix	= Path.Local.sPrefix
-			Path.App.sSBin		= Path.App.sPrefix + "/sbin"
-		# Local
-		else :
-			Path.App.sPrefix	= os.path.realpath( os.path.dirname( __file__ ) ) + "/../"
-			Path.App.sSBin		= Path.App.sPrefix + "/Bin"
-			Path.App.sInstall	= ""
+#		# Installation
+#		# KDE
+#		if QtCore.QFile.exists( Path.Sys.sPrefix + Path.App.sInstall + "/Source/" ) :
+#			Path.App.sPrefix	= Path.Sys.sPrefix
+#			Path.App.sSBin		= Path.App.sPrefix + "/sbin"
+#		# User
+#		elif QtCore.QFile.exists( Path.Local.sPrefix + Path.App.sInstall + "/Source/" ) :
+#			Path.App.sPrefix	= Path.Local.sPrefix
+#			Path.App.sSBin		= Path.App.sPrefix + "/sbin"
+#		# Local
+#		else :
+#			Path.App.sPrefix	= os.path.realpath( os.path.dirname( __file__ ) ) + "/../"
+#			Path.App.sSBin		= Path.App.sPrefix + "/Bin"
+#			Path.App.sInstall	= ""
 
 		# Proxy
 		if Env.sProxy != "" :
