diff -ur Q7Z-orig/Bin/Q7Z.pyw Q7Z/Bin/Q7Z.pyw
--- Q7Z-orig/Bin/Q7Z.pyw	2008-06-29 03:11:21.000000000 +0300
+++ Q7Z/Bin/Q7Z.pyw	2009-05-30 00:29:09.712445008 +0300
@@ -11,7 +11,7 @@
 ### Imports
 
 # Q7Z
-import	Main
+import	q7z.Main as Main
 
 
 ### Execution
diff -ur Q7Z-orig/Source/Q7Z.pyw Q7Z/Source/Q7Z.pyw
--- Q7Z-orig/Source/Q7Z.pyw	2008-06-29 03:11:21.000000000 +0300
+++ Q7Z/Source/Q7Z.pyw	2009-05-30 00:29:09.712445008 +0300
@@ -11,7 +11,7 @@
 ### Imports
 
 # Q7Z
-import	Main
+import	q7z.Main as Main
 
 
 ### Execution
