--- eric/install.py.orig	2008-11-21 14:32:48.000000000 +0100
+++ eric/install.py	2008-11-21 14:34:34.000000000 +0100
@@ -662,9 +662,6 @@
                     (v[0], v[1], v[2])
                 print "    with foreign characters. Please see 'README-PyXML.txt' for"
                 print "    details."
-                res = raw_input("    Shall pyXML be patched now (y/n)? ")
-                if res in ["Y", "y"]:
-                    patchPyXML()
     except:
         pass
     
--- install.py.orig	2008-11-21 14:43:06.000000000 +0100
+++ install.py	2008-11-21 14:43:24.000000000 +0100
@@ -665,9 +665,6 @@
                     (v[0], v[1], v[2])
                 print "    with foreign characters. Please see 'README-PyXML.txt' for"
                 print "    details."
-                res = raw_input("    Shall pyXML be patched now (y/n)? ")
-                if res in ["Y", "y"]:
-                    patchPyXML()
     except:
         pass
     
