diff -Naur eric4-4.4-snapshot-20090913~orig/install.py eric4-4.4-snapshot-20090913/install.py
--- eric4-4.4-snapshot-20090913~orig/install.py	2009-09-26 00:25:42.000000000 +0200
+++ eric4-4.4-snapshot-20090913/install.py	2009-09-26 00:28:29.000000000 +0200
@@ -672,9 +672,6 @@
                     (v[0], v[1], v[2])
                 print "    with foreign characters. Please see 'README-PyXML.txt' for"
                 print "    details."
-                res = raw_input("    Shall pyXML be patched now (y/n)? ")
-                if res in ["Y", "y"]:
-                    patchPyXML()
     except ImportError:
         pass
     
@@ -695,7 +692,6 @@
             print "See the patches directory for details."
     
     print
-    raw_input("Press enter to continue...")
     
     
 if __name__ == "__main__":
