Index: eric5-5.0-snapshot-20100523/eric/DataViews/PyCoverageDialog.py
===================================================================
--- eric5-5.0-snapshot-20100523.orig/eric/DataViews/PyCoverageDialog.py
+++ eric5-5.0-snapshot-20100523/eric/DataViews/PyCoverageDialog.py
@@ -15,7 +15,7 @@ from PyQt4.QtGui import *
 from .Ui_PyCoverageDialog import Ui_PyCoverageDialog
 
 import Utilities
-from DebugClients.Python3.coverage import coverage
+from coverage import coverage
 
 class PyCoverageDialog(QDialog, Ui_PyCoverageDialog):
     """
Index: eric5-5.0-snapshot-20100523/eric/eric5.e4p
===================================================================
--- eric5-5.0-snapshot-20100523.orig/eric/eric5.e4p
+++ eric5-5.0-snapshot-20100523/eric/eric5.e4p
@@ -542,12 +542,6 @@
     <Source>Helpviewer/HelpTopicDialog.py</Source>
     <Source>Helpviewer/HelpSearchWidget.py</Source>
     <Source>Helpviewer/HelpDocsInstaller.py</Source>
-    <Source>DebugClients/Python/coverage/data.py</Source>
-    <Source>DebugClients/Python/coverage/cmdline.py</Source>
-    <Source>DebugClients/Python/coverage/collector.py</Source>
-    <Source>DebugClients/Python/coverage/misc.py</Source>
-    <Source>DebugClients/Python/coverage/control.py</Source>
-    <Source>DebugClients/Python/coverage/__init__.py</Source>
     <Source>Debugger/DebuggerInterfacePython3.py</Source>
     <Source>Preferences/ConfigurationPages/DebuggerPython3Page.py</Source>
     <Source>DebugClients/Python3/AsyncFile.py</Source>
@@ -568,15 +562,6 @@
     <Source>Helpviewer/HelpWebSearchWidget.py</Source>
     <Source>Helpviewer/HelpClearPrivateDataDialog.py</Source>
     <Source>Helpviewer/HelpLanguagesDialog.py</Source>
-    <Source>DebugClients/Python/coverage/html.py</Source>
-    <Source>DebugClients/Python/coverage/report.py</Source>
-    <Source>DebugClients/Python/coverage/templite.py</Source>
-    <Source>DebugClients/Python/coverage/codeunit.py</Source>
-    <Source>DebugClients/Python/coverage/summary.py</Source>
-    <Source>DebugClients/Python/coverage/files.py</Source>
-    <Source>DebugClients/Python/coverage/annotate.py</Source>
-    <Source>DebugClients/Python/coverage/parser.py</Source>
-    <Source>DebugClients/Python/coverage/execfile.py</Source>
     <Source>Helpviewer/CookieJar/__init__.py</Source>
     <Source>Helpviewer/CookieJar/CookiesExceptionsDialog.py</Source>
     <Source>Helpviewer/CookieJar/CookieModel.py</Source>
@@ -631,23 +616,6 @@
     <Source>IconEditor/IconEditorWindow.py</Source>
     <Source>IconEditor/IconZoomDialog.py</Source>
     <Source>IconEditor/IconEditorPalette.py</Source>
-    <Source>DebugClients/Python/coverage/backward.py</Source>
-    <Source>DebugClients/Python3/coverage/__init__.py</Source>
-    <Source>DebugClients/Python3/coverage/annotate.py</Source>
-    <Source>DebugClients/Python3/coverage/backward.py</Source>
-    <Source>DebugClients/Python3/coverage/cmdline.py</Source>
-    <Source>DebugClients/Python3/coverage/codeunit.py</Source>
-    <Source>DebugClients/Python3/coverage/collector.py</Source>
-    <Source>DebugClients/Python3/coverage/control.py</Source>
-    <Source>DebugClients/Python3/coverage/data.py</Source>
-    <Source>DebugClients/Python3/coverage/execfile.py</Source>
-    <Source>DebugClients/Python3/coverage/files.py</Source>
-    <Source>DebugClients/Python3/coverage/html.py</Source>
-    <Source>DebugClients/Python3/coverage/misc.py</Source>
-    <Source>DebugClients/Python3/coverage/parser.py</Source>
-    <Source>DebugClients/Python3/coverage/report.py</Source>
-    <Source>DebugClients/Python3/coverage/summary.py</Source>
-    <Source>DebugClients/Python3/coverage/templite.py</Source>
     <Source>ThirdParty/Pygments/pygments/lexers/parsers.py</Source>
     <Source>Helpviewer/Passwords/__init__.py</Source>
     <Source>Helpviewer/Passwords/PasswordManager.py</Source>
@@ -700,14 +668,6 @@
     <Source>eric5-api.py</Source>
     <Source>DebugClients/Python/eric5dbgstub.py</Source>
     <Source>DebugClients/Python3/eric5dbgstub.py</Source>
-    <Source>DebugClients/Python3/coverage/bytecode.py</Source>
-    <Source>DebugClients/Python3/coverage/xmlreport.py</Source>
-    <Source>DebugClients/Python3/coverage/phystokens.py</Source>
-    <Source>DebugClients/Python3/coverage/results.py</Source>
-    <Source>DebugClients/Python/coverage/bytecode.py</Source>
-    <Source>DebugClients/Python/coverage/xmlreport.py</Source>
-    <Source>DebugClients/Python/coverage/phystokens.py</Source>
-    <Source>DebugClients/Python/coverage/results.py</Source>
     <Source>E5XML/ProjectWriter.py</Source>
     <Source>E5XML/SessionWriter.py</Source>
     <Source>E5XML/ProjectHandler.py</Source>
@@ -1087,15 +1047,11 @@
     <Other>ThirdParty/Pygments/pygments/AUTHORS</Other>
     <Other>ThirdParty/Pygments/pygments/PKG-INFO</Other>
     <Other>Documentation/Help</Other>
-    <Other>DebugClients/Python/coverage/htmlfiles</Other>
     <Other>IconEditor/cursors/aim-cursor.xpm</Other>
     <Other>IconEditor/cursors/colorpicker-cursor.xpm</Other>
     <Other>IconEditor/cursors/eraser-cursor.xpm</Other>
     <Other>IconEditor/cursors/fill-cursor.xpm</Other>
     <Other>IconEditor/cursors/paintbrush-cursor.xpm</Other>
-    <Other>DebugClients/Python/coverage/doc</Other>
-    <Other>DebugClients/Python3/coverage/doc</Other>
-    <Other>DebugClients/Python3/coverage/htmlfiles</Other>
     <Other>eric5config.linux</Other>
     <Other>eric5.e4p</Other>
     <Other>README-eric5-doc.txt</Other>
@@ -1451,4 +1407,4 @@
       </dict>
     </DocumentationParams>
   </Documentation>
-</Project>
\ No newline at end of file
+</Project>
Index: eric5-5.0-snapshot-20100523/eric/PyUnit/UnittestDialog.py
===================================================================
--- eric5-5.0-snapshot-20100523.orig/eric/PyUnit/UnittestDialog.py
+++ eric5-5.0-snapshot-20100523/eric/PyUnit/UnittestDialog.py
@@ -23,7 +23,7 @@ from E5Gui.E5Completers import E5FileCom
 from .Ui_UnittestDialog import Ui_UnittestDialog
 from .Ui_UnittestStacktraceDialog import Ui_UnittestStacktraceDialog
 
-from DebugClients.Python3.coverage import coverage
+from coverage import coverage
 
 import UI.PixmapCache
 
@@ -592,4 +592,4 @@ class UnittestWindow(QMainWindow):
             QApplication.exit()
             return True
         
-        return False
\ No newline at end of file
+        return False
Index: eric5-5.0-snapshot-20100523/eric/QScintilla/Editor.py
===================================================================
--- eric5-5.0-snapshot-20100523.orig/eric/QScintilla/Editor.py
+++ eric5-5.0-snapshot-20100523/eric/QScintilla/Editor.py
@@ -26,7 +26,7 @@ from .SpellCheckingDialog import SpellCh
 
 from Debugger.EditBreakpointDialog import EditBreakpointDialog
 
-from DebugClients.Python3.coverage import coverage
+from coverage import coverage
 
 from DataViews.CodeMetricsDialog import CodeMetricsDialog
 from DataViews.PyCoverageDialog import PyCoverageDialog
