Index: eric4-4.5-snapshot-20100523/eric/DataViews/PyCoverageDialog.py
===================================================================
--- eric4-4.5-snapshot-20100523.orig/eric/DataViews/PyCoverageDialog.py
+++ eric4-4.5-snapshot-20100523/eric/DataViews/PyCoverageDialog.py
@@ -19,7 +19,7 @@ from KdeQt.KQProgressDialog import KQPro
 from Ui_PyCoverageDialog import Ui_PyCoverageDialog
 
 import Utilities
-from DebugClients.Python.coverage import coverage
+from coverage import coverage
 
 class PyCoverageDialog(QDialog, Ui_PyCoverageDialog):
     """
Index: eric4-4.5-snapshot-20100523/eric/eric4.e4p
===================================================================
--- eric4-4.5-snapshot-20100523.orig/eric/eric4.e4p
+++ eric4-4.5-snapshot-20100523/eric/eric4.e4p
@@ -614,12 +614,6 @@
     <Source>Helpviewer/HelpTopicDialog.py</Source>
     <Source>Helpviewer/HelpSearchWidget.py</Source>
     <Source>Helpviewer/HelpDocsInstaller.py</Source>
-    <Source>DebugClients/Python/coverage/data.py</Source>
-    <Source>DebugClients/Python/coverage/cmdline.py</Source>
-    <Source>DebugClients/Python/coverage/collector.py</Source>
-    <Source>DebugClients/Python/coverage/misc.py</Source>
-    <Source>DebugClients/Python/coverage/control.py</Source>
-    <Source>DebugClients/Python/coverage/__init__.py</Source>
     <Source>Debugger/DebuggerInterfacePython3.py</Source>
     <Source>Preferences/ConfigurationPages/DebuggerPython3Page.py</Source>
     <Source>DebugClients/Python3/AsyncFile.py</Source>
@@ -652,15 +646,6 @@
     <Source>E4Network/E4NetworkMonitor.py</Source>
     <Source>E4Network/E4NetworkHeaderDetailsDialog.py</Source>
     <Source>Helpviewer/HelpLanguagesDialog.py</Source>
-    <Source>DebugClients/Python/coverage/html.py</Source>
-    <Source>DebugClients/Python/coverage/report.py</Source>
-    <Source>DebugClients/Python/coverage/templite.py</Source>
-    <Source>DebugClients/Python/coverage/codeunit.py</Source>
-    <Source>DebugClients/Python/coverage/summary.py</Source>
-    <Source>DebugClients/Python/coverage/files.py</Source>
-    <Source>DebugClients/Python/coverage/annotate.py</Source>
-    <Source>DebugClients/Python/coverage/parser.py</Source>
-    <Source>DebugClients/Python/coverage/execfile.py</Source>
     <Source>Helpviewer/CookieJar/__init__.py</Source>
     <Source>Helpviewer/CookieJar/CookiesExceptionsDialog.py</Source>
     <Source>Helpviewer/CookieJar/CookieModel.py</Source>
@@ -720,23 +705,6 @@
     <Source>eric4-iconeditor.py</Source>
     <Source>IconEditor/IconZoomDialog.py</Source>
     <Source>IconEditor/IconEditorPalette.py</Source>
-    <Source>DebugClients/Python/coverage/backward.py</Source>
-    <Source>DebugClients/Python3/coverage/__init__.py</Source>
-    <Source>DebugClients/Python3/coverage/annotate.py</Source>
-    <Source>DebugClients/Python3/coverage/backward.py</Source>
-    <Source>DebugClients/Python3/coverage/cmdline.py</Source>
-    <Source>DebugClients/Python3/coverage/codeunit.py</Source>
-    <Source>DebugClients/Python3/coverage/collector.py</Source>
-    <Source>DebugClients/Python3/coverage/control.py</Source>
-    <Source>DebugClients/Python3/coverage/data.py</Source>
-    <Source>DebugClients/Python3/coverage/execfile.py</Source>
-    <Source>DebugClients/Python3/coverage/files.py</Source>
-    <Source>DebugClients/Python3/coverage/html.py</Source>
-    <Source>DebugClients/Python3/coverage/misc.py</Source>
-    <Source>DebugClients/Python3/coverage/parser.py</Source>
-    <Source>DebugClients/Python3/coverage/report.py</Source>
-    <Source>DebugClients/Python3/coverage/summary.py</Source>
-    <Source>DebugClients/Python3/coverage/templite.py</Source>
     <Source>E4Gui/E4ModelToolBar.py</Source>
     <Source>E4Gui/E4TreeSortFilterProxyModel.py</Source>
     <Source>ThirdParty/Pygments/pygments/lexers/parsers.py</Source>
@@ -1045,15 +1013,11 @@
     <Other>ThirdParty/Pygments/pygments/PKG-INFO</Other>
     <Other>ThirdParty/SimpleJSON/simplejson/LICENSE.txt</Other>
     <Other>ThirdParty/SimpleJSON/simplejson/PKG-INFO</Other>
-    <Other>DebugClients/Python/coverage/htmlfiles</Other>
     <Other>IconEditor/cursors/aim-cursor.xpm</Other>
     <Other>IconEditor/cursors/colorpicker-cursor.xpm</Other>
     <Other>IconEditor/cursors/eraser-cursor.xpm</Other>
     <Other>IconEditor/cursors/fill-cursor.xpm</Other>
     <Other>IconEditor/cursors/paintbrush-cursor.xpm</Other>
-    <Other>DebugClients/Python/coverage/doc</Other>
-    <Other>DebugClients/Python3/coverage/doc</Other>
-    <Other>DebugClients/Python3/coverage/htmlfiles</Other>
     <Other>Documentation/Help</Other>
   </Others>
   <MainScript>eric4.py</MainScript>
@@ -1512,4 +1476,4 @@
       </dict>
     </CheckersParams>
   </Checkers>
-</Project>
\ No newline at end of file
+</Project>
Index: eric4-4.5-snapshot-20100523/eric/PyUnit/UnittestDialog.py
===================================================================
--- eric4-4.5-snapshot-20100523.orig/eric/PyUnit/UnittestDialog.py
+++ eric4-4.5-snapshot-20100523/eric/PyUnit/UnittestDialog.py
@@ -26,7 +26,7 @@ from E4Gui.E4Completers import E4FileCom
 from Ui_UnittestDialog import Ui_UnittestDialog
 from Ui_UnittestStacktraceDialog import Ui_UnittestStacktraceDialog
 
-from DebugClients.Python.coverage import coverage
+from coverage import coverage
 
 import UI.PixmapCache
 
Index: eric4-4.5-snapshot-20100523/eric/QScintilla/Editor.py
===================================================================
--- eric4-4.5-snapshot-20100523.orig/eric/QScintilla/Editor.py
+++ eric4-4.5-snapshot-20100523/eric/QScintilla/Editor.py
@@ -28,7 +28,7 @@ from SpellCheckingDialog import SpellChe
 
 from Debugger.EditBreakpointDialog import EditBreakpointDialog
 
-from DebugClients.Python.coverage import coverage
+from coverage import coverage
 
 from DataViews.CodeMetricsDialog import CodeMetricsDialog
 from DataViews.PyCoverageDialog import PyCoverageDialog
