--- eric5-5.0.0~orig/install.py	2010-07-06 01:02:54.000000000 +0200
+++ eric5-5.0.0/install.py	2010-07-06 01:02:39.000000000 +0200
@@ -408,11 +408,6 @@
                     shutil.copy(apiName, apidir)
                 except EnvironmentError:
                     print("Could not install '%s'." % apiName)
-    
-    # create menu entry for Linux systems
-    if sys.platform.startswith("linux"):
-        shutil.copy(os.path.join("eric", "icons", "default", "eric.png"), "/usr/share/pixmaps")
-        shutil.copy(os.path.join("eric", "eric5.desktop"), "/usr/share/applications")
 
 
 def createInstallConfig():
