Index: eric4-4.4-snapshot-20090627/install.py
===================================================================
--- eric4-4.4-snapshot-20090627.orig/install.py
+++ eric4-4.4-snapshot-20090627/install.py
@@ -668,16 +668,9 @@ def main(argv):
                     (v[0], v[1], v[2])
                 print "    with foreign characters. Please see 'README-PyXML.txt' for"
                 print "    details."
-                res = raw_input("    Shall pyXML be patched now (y/n)? ")
-                if res in ["Y", "y"]:
-                    patchPyXML()
     except ImportError:
         pass
     
-    print
-    raw_input("Press enter to continue...")
-    
-    
 if __name__ == "__main__":
     try:
         main(sys.argv)
