diff --git a/converseen.pro b/converseen.pro
index dfe35c4..9cf07b4 100755
--- a/converseen.pro
+++ b/converseen.pro
@@ -29,11 +29,6 @@ CONFIG += qt \
   src/dialogoptions.cpp \
   src/whereiam.cpp \
   src/dialoginfo.cpp
- QMAKE_CXXFLAGS +=  -g \
-  -O2 \
-  -Wall \
-  -W \
-  -pthread
  INCLUDEPATH +=  /usr/include/ImageMagick
  LIBS +=  -L/usr/lib  -lMagick++  -lMagickWand  -lMagickCore
  TRANSLATIONS +=  loc/converseen_it_IT.ts
