--- src/src.pro.old	2009-05-25 00:55:37.369064692 +0300
+++ src/src.pro	2009-05-25 00:56:10.167178104 +0300
@@ -6,7 +6,7 @@
 TARGET = okindd
 DESTDIR = ../bin
 HEADERS += okindadapter.h
-INSTALLS += target pics examples shortcut
+INSTALLS += target pics examples
 
 target.path = /usr/bin/
 
@@ -16,6 +16,3 @@
 examples.path = /usr/share/doc/okindd/examples/
 examples.files += ../scripts/* ../conf/okinddrc.example
 
-shortcut.path = ~/.kde/Autostart/
-shortcut.files += ../okindd.desktop
-
