--- src/src.pro	2009-02-24 22:44:59.000000000 +0200
+++ src/src.pro.new	2009-03-17 22:11:15.580885471 +0200
@@ -6,19 +6,15 @@
 TARGET = okindd
 DESTDIR = ../bin
 HEADERS += okindadapter.h
-INSTALLS += target  pics examples config shortcut
+INSTALLS += target  pics examples config
 
 target.path = /usr/bin/
 
 pics.path = /usr/share/okindd/
 pics.files += ../pics/*
 
-examples.path = /usr/share/doc/okind/examples/
+examples.path = /usr/share/doc/okindd/examples/
 examples.files += ../scripts/*
 
-config.path = ~/.okind/
+config.path = /usr/share/doc/okindd/
 config.files += ../conf/okinddrc.example
-
-shortcut.path = ~/.kde/Autostart/
-shortcut.files += ../okindd.desktop
-
--- src/okindadapter.cpp	2009-02-24 22:44:59.000000000 +0200
+++ src/okindadapter.cpp	2009-03-18 13:48:52.954121009 +0200
@@ -88,7 +88,7 @@
 // Выбор файла конфигурации и загрузка массива из файла конфигурации
 void OkindAdapter::setConfigFile (QString confFile){
 	configFile=confFile;
-	if (confFile=="") configFile=HomeDir+"/.okind/okinddrc";
+	if (confFile=="") configFile=HomeDir+"/.okindd/okinddrc";
 	QSettings cfg (configFile, QSettings::NativeFormat);
 	QStringList groups = cfg.childGroups();
 	profiles.clear();
