--- imapquickcheck.desktop
+++ imapquickcheck.desktop
@@ -1,8 +1,6 @@
 [Desktop Entry]
-Categories=Network;QT;
-Comment[fr]=Vérificateur de Courrier IMAP
-Comment=IMAP Mailbox Checker
-Encoding=UTF-8
+Categories=Qt;Email;Network;Office;
+Comment=Lightweight systray IMAP mail checker
 Exec=imapquickcheck
 GenericName[fr]=Vérificateur de Courrier IMAP
 GenericName=IMAP Mailbox Checker
@@ -10,12 +8,10 @@
 MimeType=text/plain;
 Name[fr]=ImapQuickCheck
 Name=ImapQuickCheck
-Path=
 StartupNotify=false
 Terminal=false
-TerminalOptions=
 Type=Application
-Version=1.3
+Version=1.0
 X-DBUS-ServiceName=
 X-DBUS-StartupType=
 X-DCOP-ServiceType=
