Index: surfer-surfer/src/bookmarksmanager.cpp
===================================================================
--- surfer-surfer.orig/src/bookmarksmanager.cpp
+++ surfer-surfer/src/bookmarksmanager.cpp
@@ -239,7 +239,7 @@ QMimeData * BookmarksManager::mimeData(c
     {
         if(index.isValid() && !index.column())
         {
-            text += QString::number(reinterpret_cast<int>(item(index))) + "\n";
+            text += QString::number(reinterpret_cast<intptr_t>(item(index))) + "\n";
         }
     }
     mimeData->setText(text);
