--- src/3rdparty/webkit/JavaScriptCore/jit/ExecutableAllocator.h	2008-12-17 17:16:38.000000000 +0000
+++ src/3rdparty/webkit/JavaScriptCore/jit/ExecutableAllocator_new.h	2008-12-24 01:01:58.000000000 +0000
@@ -59,7 +59,7 @@
         if (n < static_cast<size_t>(m_end - m_freePtr)) {
             char* result = m_freePtr;
             // ensure m_freePtr is word aligned.
-            m_freePtr += n + (sizeof(void*) - n & (sizeof(void*) - 1));
+            m_freePtr += n + ((sizeof(void*) - n) & (sizeof(void*) - 1));
             return result;
         }
 
