--- qt-x11-opensource-src-4.5.9999.orig/src/3rdparty/webkit/WebCore/WebCore.pro	2009-07-23 21:43:38.532334796 +0200
+++ qt-x11-opensource-src-4.5.9999/src/3rdparty/webkit/WebCore/WebCore.pro	2009-07-23 21:57:20.547084194 +0200
@@ -2,6 +2,8 @@
 CONFIG += building-libs
 CONFIG += depend_includepath
 include($$PWD/../WebKit.pri)
+# include media-libs/phonon headers
+INCLUDEPATH += /usr/include
 
 TEMPLATE = lib
 TARGET = QtWebKit
--- qt-x11-opensource-src-4.5.9999.orig/src/3rdparty/webkit/WebCore/platform/graphics/qt/MediaPlayerPrivatePhonon.cpp	2009-07-23 21:43:38.899333887 +0200
+++ qt-x11-opensource-src-4.5.9999/src/3rdparty/webkit/WebCore/platform/graphics/qt/MediaPlayerPrivatePhonon.cpp	2009-07-23 21:59:58.569084921 +0200
@@ -36,9 +36,11 @@
 #include <QUrl>
 #include <QEvent>
 
-#include <Phonon/AudioOutput>
-#include <Phonon/MediaObject>
-#include <Phonon/VideoWidget>
+#include <phonon/mediaobject.h>
+#include <phonon/mediasource.h>
+#include <phonon/videowidget.h>
+#include <phonon/audiooutput.h>
+#include <phonon/medianode.h>
 
 using namespace Phonon;
 
