--- PyQt-x11-gpl-4.6.1~orig/configure.py	2009-10-23 23:03:13.000000000 +0200
+++ PyQt-x11-gpl-4.6.1/configure.py	2009-10-23 23:33:06.000000000 +0200
@@ -341,7 +341,7 @@
         check_module("QtWebKit", "qwebpage.h", "new QWebPage()")
         check_module("QtXml", "qdom.h", "new QDomDocument()")
         check_module("QtXmlPatterns", "qxmlname.h", "new QXmlName()")
-        check_module("phonon", "Phonon/VideoWidget",
+        check_module("phonon", "phonon/videowidget.h",
                 "new Phonon::VideoWidget()")
         check_module("QtAssistant", "qassistantclient.h",
                 "new QAssistantClient(\"foo\")", extra_lib_dirs=ass_lib_dirs,
@@ -451,7 +451,7 @@
             generate_code("QtXmlPatterns")
 
         if "phonon" in pyqt_modules:
-            generate_code("phonon")
+            generate_code("phonon", extra_include_dirs=["/usr/include/phonon"])
 
         if "QtAssistant" in pyqt_modules:
             generate_code("QtAssistant")
@@ -759,6 +759,7 @@
                 install_dir=pydbusmoddir,
                 qt=["QtCore"],
                 debug=opts.debug,
+                strip=0,
                 universal=sipcfg.universal,
                 arch=sipcfg.arch
             )
@@ -1468,6 +1469,7 @@
         warnings=1,
         static=opts.static,
         debug=opts.debug,
+        strip=0,
         universal=sipcfg.universal,
         arch=sipcfg.arch
     )
