diff --git a/PySide/QtScriptTools/CMakeLists.txt b/PySide/QtScriptTools/CMakeLists.txt
index 18e1f93..ffcd890 100644
--- a/PySide/QtScriptTools/CMakeLists.txt
+++ b/PySide/QtScriptTools/CMakeLists.txt
@@ -21,7 +21,7 @@ if(NOT QT_QTSCRIPTTOOLS_LIBRARY)
   if(CMAKE_HOST_APPLE)
     set(QT_QTSCRIPTTOOLS_LIBRARY "-framework QtScriptTools")
   else(CMAKE_HOST_APPLE)
-    set(QT_QTSCRIPTTOOLS_LIBRARY -lQtScriptTools)
+    set(QT_QTSCRIPTTOOLS_LIBRARY ${QT_QTSCRIPT_LIBRARY}Tools)
   endif(CMAKE_HOST_APPLE)
 endif(NOT QT_QTSCRIPTTOOLS_LIBRARY)
 
