diff -ru qt4-qtruby-2.0.3.original/ruby/qttest/CMakeLists.txt qt4-qtruby-2.0.3.patched/ruby/qttest/CMakeLists.txt
--- qt4-qtruby-2.0.3.original/ruby/qttest/CMakeLists.txt	2008-11-04 12:24:26.000000000 +0100
+++ qt4-qtruby-2.0.3.patched/ruby/qttest/CMakeLists.txt	2009-04-23 18:11:44.696915231 +0200
@@ -4,7 +4,7 @@
 set(qttest_LIBRARY_SRC qttest.cpp qttesthandlers.cpp)
 
 add_library(qttest MODULE ${qttest_LIBRARY_SRC})
-target_link_libraries(qttest ${QT_QTCORE_LIBRARY} ${RUBY_LIBRARY} smokeqt smokekde smokeqttest qtruby4shared)
+target_link_libraries(qttest ${QT_QTCORE_LIBRARY} ${RUBY_LIBRARY} smokeqt smokeqttest qtruby4shared)
 set_target_properties(qttest PROPERTIES PREFIX "")
 install(TARGETS qttest DESTINATION ${CUSTOM_RUBY_SITE_ARCH_DIR})
 install(FILES qttest.rb DESTINATION ${CUSTOM_RUBY_SITE_LIB_DIR}/qttest)
