diff -Naur QScintilla-gpl-2.4.orig/designer-Qt3/designer.pro QScintilla-gpl-2.4/designer-Qt3/designer.pro
--- QScintilla-gpl-2.4.orig/designer-Qt3/designer.pro	2009-06-07 21:55:35.000000000 +0200
+++ QScintilla-gpl-2.4/designer-Qt3/designer.pro	2009-06-07 22:01:14.000000000 +0200
@@ -3,10 +3,13 @@
 
 TEMPLATE = lib
 TARGET = qscintillaplugin
-DESTDIR = $(QTDIR)/plugins/designer
 
 CONFIG += qt warn_on release plugin
 
 SOURCES += qscintillaplugin.cpp
 
-LIBS += -lqscintilla2
+target.path = $(QTDIR)/plugins/designer
+INSTALLS += target
+
+INCLUDEPATH = ../Qt3
+LIBS += -L../Qt3 -lqscintilla2
diff -Naur QScintilla-gpl-2.4.orig/designer-Qt4/designer.pro QScintilla-gpl-2.4/designer-Qt4/designer.pro
--- QScintilla-gpl-2.4.orig/designer-Qt4/designer.pro	2009-06-07 21:55:35.000000000 +0200
+++ QScintilla-gpl-2.4/designer-Qt4/designer.pro	2009-06-07 21:57:46.000000000 +0200
@@ -12,4 +12,5 @@
 target.path = $$[QT_INSTALL_PLUGINS]/designer
 INSTALLS += target
 
-LIBS += -lqscintilla2
+INCLUDEPATH = ../Qt4
+LIBS += -L../Qt4 -lqscintilla2
