--- designer.pro.ori	2006-12-13 09:31:54.000000000 +0700
+++ designer.pro	2006-12-13 09:40:49.000000000 +0700
@@ -4,12 +4,14 @@
 TEMPLATE = lib
 TARGET = qscintillaplugin
 
+INCLUDEPATH = ../Qt4
+
 CONFIG += designer release plugin
 
 HEADERS = qscintillaplugin.h
 SOURCES = qscintillaplugin.cpp
 
-target.path = $$[QT_INSTALL_PLUGINS]/designer
+target.path = .
 INSTALLS += target
 
-LIBS += -lqscintilla2
+LIBS += -L../Qt4 -lqscintilla2
