diff --git a/cmake/Ts.cmake b/cmake/Ts.cmake
index e40f078..5b7ab87 100644
--- a/cmake/Ts.cmake
+++ b/cmake/Ts.cmake
@@ -59,7 +59,7 @@ MACRO(COMPILE_AVIDEMUX_TS_FILES ts_subdir _sources)
                 COMMAND ${LRELEASE_EXECUTABLE}
                     ${_outXml}
                     -qm ${_out}
-                DEPENDS ${_in}
+                DEPENDS ${_outXml}
             )
                 
             SET(qm_files ${qm_files} ${_outXml} ${_out})
