diff --git a/avidemux/ADM_UIs/ADM_CLI/src/CMakeLists.txt b/avidemux/ADM_UIs/ADM_CLI/src/CMakeLists.txt
index 16a75a2..826d9aa 100644
--- a/avidemux/ADM_UIs/ADM_CLI/src/CMakeLists.txt
+++ b/avidemux/ADM_UIs/ADM_CLI/src/CMakeLists.txt
@@ -9,4 +9,4 @@ SET(${ADM_LIB}_SRCS
 INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/../include)
 ADD_LIBRARY(${ADM_LIB}  SHARED ${${ADM_LIB}_SRCS})
 ADD_SOURCE_CFLAGS(DIA_flyDialogCli.cpp " -DADM_UI_TYPE_BUILD=1")
-INSTALL(TARGETS ${ADM_LIB} RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib  ARCHIVE DESTINATION lib)
+INSTALL(TARGETS ${ADM_LIB} RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib${LIB_SUFFIX}  ARCHIVE DESTINATION lib${LIB_SUFFIX})
diff --git a/avidemux/ADM_UIs/ADM_GTK/src/CMakeLists.txt b/avidemux/ADM_UIs/ADM_GTK/src/CMakeLists.txt
index 3f2f26f..cf2640f 100644
--- a/avidemux/ADM_UIs/ADM_GTK/src/CMakeLists.txt
+++ b/avidemux/ADM_UIs/ADM_GTK/src/CMakeLists.txt
@@ -30,4 +30,4 @@ ADD_LIBRARY(${ADM_LIB} SHARED ${${ADM_LIB}_SRCS})
 TARGET_LINK_LIBRARIES(${ADM_LIB} ${GTK_LDFLAGS} ADM_core ADM_coreUI ADM_render_gtk)
 ADD_TARGET_CFLAGS(${ADM_LIB} ${GTK_CFLAGS})
 ADD_SOURCE_CFLAGS(DIA_flyDialogGtk.cpp " -DADM_UI_TYPE_BUILD=2")
-INSTALL(TARGETS ${ADM_LIB} RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib  ARCHIVE DESTINATION lib)
\ No newline at end of file
+INSTALL(TARGETS ${ADM_LIB} RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib${LIB_SUFFIX}  ARCHIVE DESTINATION lib${LIB_SUFFIX})
diff --git a/avidemux/ADM_UIs/ADM_QT4/src/CMakeLists.txt b/avidemux/ADM_UIs/ADM_QT4/src/CMakeLists.txt
index 8e1a46f..b717bb1 100644
--- a/avidemux/ADM_UIs/ADM_QT4/src/CMakeLists.txt
+++ b/avidemux/ADM_UIs/ADM_QT4/src/CMakeLists.txt
@@ -21,4 +21,4 @@ ADD_LIBRARY(${ADM_LIB} SHARED ${${ADM_LIB}_SRCS})
 ADD_SOURCE_CFLAGS(T_flyDialogQt4.cpp " -DADM_UI_TYPE_BUILD=4")
 ADD_SOURCE_CFLAGS(T_QCanvas.cpp " -DADM_UI_TYPE_BUILD=4")
 TARGET_LINK_LIBRARIES(${ADM_LIB} ADM_core ADM_coreUI ${QT_QTGUI_LIBRARY} ${QT_QTCORE_LIBRARY} ADM_render_qt4)
-INSTALL(TARGETS ${ADM_LIB} RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib  ARCHIVE DESTINATION lib)
+INSTALL(TARGETS ${ADM_LIB} RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib${LIB_SUFFIX}  ARCHIVE DESTINATION lib${LIB_SUFFIX})
diff --git a/avidemux/ADM_core/src/CMakeLists.txt b/avidemux/ADM_core/src/CMakeLists.txt
index b77489a..072a67b 100644
--- a/avidemux/ADM_core/src/CMakeLists.txt
+++ b/avidemux/ADM_core/src/CMakeLists.txt
@@ -20,4 +20,4 @@ ENDIF (MINGW)
 REMOVE_DEFINITIONS(-DHAVE_CONFIG_H)
 include_directories("${FFMPEG_BINARY_DIR}")
 
-INSTALL(TARGETS ADM_core RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib  ARCHIVE DESTINATION lib)
+INSTALL(TARGETS ADM_core RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib${LIB_SUFFIX}  ARCHIVE DESTINATION lib${LIB_SUFFIX})
diff --git a/avidemux/ADM_coreAudio/src/CMakeLists.txt b/avidemux/ADM_coreAudio/src/CMakeLists.txt
index 2a62145..c969241 100644
--- a/avidemux/ADM_coreAudio/src/CMakeLists.txt
+++ b/avidemux/ADM_coreAudio/src/CMakeLists.txt
@@ -17,4 +17,4 @@ ADD_LIBRARY(ADM_coreAudio SHARED ${ADMaudioCore_SRCS})
 INCLUDE_DIRECTORIES(${CMAKE_SOURCE_DIR}/avidemux/ADM_coreAudio/include)
 TARGET_LINK_LIBRARIES(ADM_coreAudio ADM_core)
 
-INSTALL(TARGETS ADM_coreAudio RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib  ARCHIVE DESTINATION lib)
+INSTALL(TARGETS ADM_coreAudio RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib${LIB_SUFFIX}  ARCHIVE DESTINATION lib${LIB_SUFFIX})
diff --git a/avidemux/ADM_coreImage/src/CMakeLists.txt b/avidemux/ADM_coreImage/src/CMakeLists.txt
index d51fb8e..f82d3d4 100644
--- a/avidemux/ADM_coreImage/src/CMakeLists.txt
+++ b/avidemux/ADM_coreImage/src/CMakeLists.txt
@@ -27,4 +27,4 @@ ENDIF (APPLE)
 
 TARGET_LINK_LIBRARIES(ADM_coreImage ADM_core ADM_coreUI ADM_libswscale ADM_libpostproc)
 add_dependencies(ADM_coreImage ffmpeg)
-INSTALL(TARGETS ADM_coreImage RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib  ARCHIVE DESTINATION lib)
+INSTALL(TARGETS ADM_coreImage RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib${LIB_SUFFIX}  ARCHIVE DESTINATION lib${LIB_SUFFIX})
diff --git a/avidemux/ADM_coreUI/src/CMakeLists.txt b/avidemux/ADM_coreUI/src/CMakeLists.txt
index 6fc5a02..c41bf47 100644
--- a/avidemux/ADM_coreUI/src/CMakeLists.txt
+++ b/avidemux/ADM_coreUI/src/CMakeLists.txt
@@ -7,4 +7,4 @@ SET(ADM_coreUI_SRCS
 ADD_LIBRARY(ADM_coreUI SHARED ${ADM_coreUI_SRCS})
 TARGET_LINK_LIBRARIES(ADM_coreUI ADM_core)
 
-INSTALL(TARGETS ADM_coreUI RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib  ARCHIVE DESTINATION lib)
+INSTALL(TARGETS ADM_coreUI RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib${LIB_SUFFIX}  ARCHIVE DESTINATION lib${LIB_SUFFIX})
diff --git a/avidemux/ADM_libraries/ADM_smjs/CMakeLists.txt b/avidemux/ADM_libraries/ADM_smjs/CMakeLists.txt
index 34d05ec..80b0d0f 100644
--- a/avidemux/ADM_libraries/ADM_smjs/CMakeLists.txt
+++ b/avidemux/ADM_libraries/ADM_smjs/CMakeLists.txt
@@ -16,4 +16,4 @@ ENDIF (MINGW)
 ADD_TARGET_CFLAGS(${ADM_LIB} "-DNDEBUG -DTRIMMED -DEXPORT_JS_API -DJS_USE_SAFE_ARENA -D_IEEE_LIBM")
 TARGET_LINK_LIBRARIES(${ADM_LIB} m)
 
-INSTALL(TARGETS ${ADM_LIB} RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib  ARCHIVE DESTINATION lib)
\ No newline at end of file
+INSTALL(TARGETS ${ADM_LIB} RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib${LIB_SUFFIX}  ARCHIVE DESTINATION lib${LIB_SUFFIX})
diff --git a/avidemux/ADM_userInterfaces/ADM_render/CMakeLists.txt b/avidemux/ADM_userInterfaces/ADM_render/CMakeLists.txt
index 42f8fb7..b4ca067 100644
--- a/avidemux/ADM_userInterfaces/ADM_render/CMakeLists.txt
+++ b/avidemux/ADM_userInterfaces/ADM_render/CMakeLists.txt
@@ -31,7 +31,7 @@ IF (ADM_UI_GTK)
 		TARGET_LINK_LIBRARIES(${ADM_LIB}_gtk ${XVIDEO_LIBRARY_DIR} X11 Xext)
 	ENDIF (USE_XV)
 
-	INSTALL(TARGETS ${ADM_LIB}_gtk RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib  ARCHIVE DESTINATION lib)
+	INSTALL(TARGETS ${ADM_LIB}_gtk RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib${LIB_SUFFIX}  ARCHIVE DESTINATION lib${LIB_SUFFIX})
 ENDIF (ADM_UI_GTK)
 
 IF (ADM_UI_QT4)
@@ -43,11 +43,11 @@ IF (ADM_UI_QT4)
 		TARGET_LINK_LIBRARIES(${ADM_LIB}_qt4 ${XVIDEO_LIBRARY_DIR} X11 Xext)
 	ENDIF (USE_XV)
 
-	INSTALL(TARGETS ${ADM_LIB}_qt4 RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib  ARCHIVE DESTINATION lib)
+	INSTALL(TARGETS ${ADM_LIB}_qt4 RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib${LIB_SUFFIX}  ARCHIVE DESTINATION lib${LIB_SUFFIX})
 ENDIF (ADM_UI_QT4)
 
 IF (GETTEXT_FOUND)
 	TARGET_LINK_LIBRARIES(${ADM_LIB}_cli ${GETTEXT_LIBRARY_DIR})
 ENDIF (GETTEXT_FOUND)
 
-INSTALL(TARGETS ${ADM_LIB}_cli RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib  ARCHIVE DESTINATION lib)
\ No newline at end of file
+INSTALL(TARGETS ${ADM_LIB}_cli RUNTIME DESTINATION ${BIN_DIR}  LIBRARY DESTINATION lib${LIB_SUFFIX}  ARCHIVE DESTINATION lib${LIB_SUFFIX})
diff --git a/avidemux/CMakeLists.txt b/avidemux/CMakeLists.txt
index f215436..59ddfe0 100644
--- a/avidemux/CMakeLists.txt
+++ b/avidemux/CMakeLists.txt
@@ -499,5 +499,5 @@ IF (APPLE)
 		SET (MAC_BUNDLE_DIR "/Applications")
 	ENDIF (MAC_BUNDLE_DIR)
 ELSE (APPLE)
-	INSTALL(TARGETS ${ADM_EXES} RUNTIME DESTINATION ${BIN_DIR} LIBRARY DESTINATION lib)
+	INSTALL(TARGETS ${ADM_EXES} RUNTIME DESTINATION ${BIN_DIR} LIBRARY DESTINATION lib${LIB_SUFFIX})
 ENDIF (APPLE)
diff --git a/cmake/ad_plugin.cmake b/cmake/ad_plugin.cmake
index 5070e18..93cf8ae 100644
--- a/cmake/ad_plugin.cmake
+++ b/cmake/ad_plugin.cmake
@@ -1,7 +1,7 @@
 IF (WIN32)
 	SET(AUDDEC_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/plugins/audioDecoder/")
 ELSE (WIN32)
-	SET(AUDDEC_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/ADM_plugins/audioDecoder/")
+	SET(AUDDEC_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/ADM_plugins/audioDecoder/")
 ENDIF (WIN32)
 
 MACRO(INIT_AUDIO_PLUGIN _lib)
diff --git a/cmake/admFFmpegBuild.cmake b/cmake/admFFmpegBuild.cmake
index 7128cd0..a150b5d 100644
--- a/cmake/admFFmpegBuild.cmake
+++ b/cmake/admFFmpegBuild.cmake
@@ -132,7 +132,7 @@ getFfmpegLibNames("${FFMPEG_SOURCE_DIR}")
 if (WIN32)
 	set(FFMPEG_INSTALL_DIR ${BIN_DIR})
 else (WIN32)
-	set(FFMPEG_INSTALL_DIR lib)
+	set(FFMPEG_INSTALL_DIR lib${LIB_SUFFIX})
 endif (WIN32)
 
 add_library(ADM_libswscale UNKNOWN IMPORTED)
diff --git a/cmake/admPluginVideoEncoder.cmake b/cmake/admPluginVideoEncoder.cmake
index e839054..387e34f 100644
--- a/cmake/admPluginVideoEncoder.cmake
+++ b/cmake/admPluginVideoEncoder.cmake
@@ -1,7 +1,7 @@
 IF (WIN32)
 	SET(VIDENC_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/plugins/videoEncoder/")
 ELSE (WIN32)
-	SET(VIDENC_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/ADM_plugins/videoEncoder/")
+	SET(VIDENC_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/ADM_plugins/videoEncoder/")
 ENDIF (WIN32)
 
 MACRO(INIT_VIDEO_ENCODER_PLUGIN _lib)
diff --git a/cmake/ae_plugin.cmake b/cmake/ae_plugin.cmake
index 83ba00b..0da9b55 100644
--- a/cmake/ae_plugin.cmake
+++ b/cmake/ae_plugin.cmake
@@ -1,7 +1,7 @@
 IF (WIN32)
 	SET(AUDENC_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/plugins/audioEncoders/")
 ELSE (WIN32)
-	SET(AUDENC_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/ADM_plugins/audioEncoders/")
+	SET(AUDENC_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/ADM_plugins/audioEncoders/")
 ENDIF (WIN32)
 
 MACRO(INIT_AUDIO_ENCODER _lib)
diff --git a/cmake/av_plugin.cmake b/cmake/av_plugin.cmake
index 61c3a40..956a959 100644
--- a/cmake/av_plugin.cmake
+++ b/cmake/av_plugin.cmake
@@ -1,7 +1,7 @@
 IF (WIN32)
 	SET(AUDDEV_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/plugins/audioDevices/")
 ELSE (WIN32)
-	SET(AUDDEV_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/ADM_plugins/audioDevices/")
+	SET(AUDDEV_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/ADM_plugins/audioDevices/")
 ENDIF (WIN32)
 
 MACRO(INIT_AUDIO_DEVICE _lib)
diff --git a/cmake/vf_plugin.cmake b/cmake/vf_plugin.cmake
index 105b65b..a55b7eb 100644
--- a/cmake/vf_plugin.cmake
+++ b/cmake/vf_plugin.cmake
@@ -1,7 +1,7 @@
 IF (WIN32)
 	SET(VIDFLT_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/plugins/videoFilter/")
 ELSE (WIN32)
-	SET(VIDFLT_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/ADM_plugins/videoFilter/")
+	SET(VIDFLT_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/ADM_plugins/videoFilter/")
 ENDIF (WIN32)
 
 MACRO(INIT_VIDEOFILTER_PLUGIN _lib)
diff --git a/plugins/ADM_videoEncoder/ADM_vidEnc_avcodec/CMakeLists.txt b/plugins/ADM_videoEncoder/ADM_vidEnc_avcodec/CMakeLists.txt
index 72eae2a..a19227d 100644
--- a/plugins/ADM_videoEncoder/ADM_vidEnc_avcodec/CMakeLists.txt
+++ b/plugins/ADM_videoEncoder/ADM_vidEnc_avcodec/CMakeLists.txt
@@ -5,7 +5,7 @@ INCLUDE(admFFmpegUtil)
 if (WIN32)
 	set(FFMPEG_INSTALL_DIR ${AVIDEMUX_INSTALL_PREFIX})
 else (WIN32)
-	set(FFMPEG_INSTALL_DIR ${AVIDEMUX_INSTALL_PREFIX}/lib)
+	set(FFMPEG_INSTALL_DIR ${AVIDEMUX_INSTALL_PREFIX}/lib${LIB_SUFFIX})
 endif (WIN32)
 
 SET(ADM_vidEnc_avcodec_SRCS  interface.c  encoder.cpp  huffyuvEncoder.cpp
@@ -29,4 +29,4 @@ TARGET_LINK_LIBRARIES(ADM_vidEnc_avcodec ${LIBXML2_LIBRARIES} ADM_core ADM_coreU
 
 INIT_VIDEO_ENCODER_PLUGIN(ADM_vidEnc_avcodec)
 INSTALL_VIDEO_ENCODER(ADM_vidEnc_avcodec)
-INSTALL(FILES Mpeg1Param.xsd DESTINATION "${VIDENC_INSTALL_DIR}${PLUGIN_SCHEMA_DIR}")
\ No newline at end of file
+INSTALL(FILES Mpeg1Param.xsd DESTINATION "${VIDENC_INSTALL_DIR}${PLUGIN_SCHEMA_DIR}")
diff --git a/plugins/CMakeLists.txt b/plugins/CMakeLists.txt
index 5c06dad..e286f8b 100644
--- a/plugins/CMakeLists.txt
+++ b/plugins/CMakeLists.txt
@@ -43,7 +43,7 @@ ELSE (NOT AVIDEMUX_INSTALL_PREFIX)
 	SET(AVIDEMUX_INSTALL_PREFIX "${AVIDEMUX_INSTALL_PREFIX}" CACHE STRING "")
 ENDIF (NOT AVIDEMUX_INSTALL_PREFIX)
 
-LINK_DIRECTORIES(${AVIDEMUX_INSTALL_PREFIX}/lib)
+LINK_DIRECTORIES(${AVIDEMUX_INSTALL_PREFIX}/lib${LIB_SUFFIX})
 
 ########################################
 # Where is coreConfig ?
@@ -98,7 +98,7 @@ IF (WIN32)
 	ENDIF (CMAKE_BUILD_TYPE STREQUAL "Release")
 ELSE (WIN32)
 	SET(BIN_DIR bin)
-	set(FFMPEG_LIB_DIR ${AVIDEMUX_INSTALL_PREFIX}/lib)
+	set(FFMPEG_LIB_DIR ${AVIDEMUX_INSTALL_PREFIX}/lib${LIB_SUFFIX})
 ENDIF (WIN32)
 
 ########################################
