diff --git a/cmake/admCheckOpencore.cmake b/cmake/admCheckOpencore.cmake
index cbd7369..3e4ed14 100644
--- a/cmake/admCheckOpencore.cmake
+++ b/cmake/admCheckOpencore.cmake
@@ -6,7 +6,7 @@ MACRO(checkOpencoreAmrnb)
 		MESSAGE(STATUS "***************************")
 
 		IF (OPENCORE_AMRNB)
-			FIND_HEADER_AND_LIB(OPENCORE_AMRNB opencore-amrnb/interf_dec.h opencore-amrnb Decoder_Interface_init)
+			FIND_HEADER_AND_LIB(OPENCORE_AMRNB amrnb/interf_dec.h amrnb Decoder_Interface_init)
 			PRINT_LIBRARY_INFO("opencore-amrnb" OPENCORE_AMRNB_FOUND "${OPENCORE_AMRNB_INCLUDE_DIR}" "${OPENCORE_AMRNB_LIBRARY_DIR}")
 		ELSE (OPENCORE_AMRNB)
 			MESSAGE("${MSG_DISABLE_OPTION}")
@@ -27,7 +27,7 @@ MACRO(checkOpencoreAmrwb)
 		MESSAGE(STATUS "***************************")
 
 		IF (OPENCORE_AMRWB)
-			FIND_HEADER_AND_LIB(OPENCORE_AMRWB opencore-amrwb/dec_if.h opencore-amrwb D_IF_init)
+			FIND_HEADER_AND_LIB(OPENCORE_AMRWB amrwb/dec_if.h amrwb D_IF_init)
 			PRINT_LIBRARY_INFO("opencore-amrwb" OPENCORE_AMRWB_FOUND "${OPENCORE_AMRWB_INCLUDE_DIR}" "${OPENCORE_AMRWB_LIBRARY_DIR}")
 		ELSE (OPENCORE_AMRWB)
 			MESSAGE("${MSG_DISABLE_OPTION}")
@@ -38,4 +38,4 @@ MACRO(checkOpencoreAmrwb)
 		
 		SET(OPENCORE_AMRWB_CHECKED 1)
 	ENDIF (NOT OPENCORE_AMRWB_CHECKED)
-ENDMACRO(checkOpencoreAmrwb)
\ No newline at end of file
+ENDMACRO(checkOpencoreAmrwb)
diff --git a/plugins/ADM_audioDecoders/ADM_ad_opencore_amrnb/ADM_ad_opencore_amrnb.cpp b/plugins/ADM_audioDecoders/ADM_ad_opencore_amrnb/ADM_ad_opencore_amrnb.cpp
index 68091ce..8e79614 100644
--- a/plugins/ADM_audioDecoders/ADM_ad_opencore_amrnb/ADM_ad_opencore_amrnb.cpp
+++ b/plugins/ADM_audioDecoders/ADM_ad_opencore_amrnb/ADM_ad_opencore_amrnb.cpp
@@ -19,7 +19,7 @@
 #include "ADM_audioCodecEnum.h"
 
 extern "C" {
-#include <opencore-amrnb/interf_dec.h>
+#include <amrnb/interf_dec.h>
 };
 
 #define ADM_AMR_BUFFER (16 * 1024) // 16 kB internal
diff --git a/plugins/ADM_audioDecoders/ADM_ad_opencore_amrwb/ADM_ad_opencore_amrwb.cpp b/plugins/ADM_audioDecoders/ADM_ad_opencore_amrwb/ADM_ad_opencore_amrwb.cpp
index 05612c9..d13c25c 100644
--- a/plugins/ADM_audioDecoders/ADM_ad_opencore_amrwb/ADM_ad_opencore_amrwb.cpp
+++ b/plugins/ADM_audioDecoders/ADM_ad_opencore_amrwb/ADM_ad_opencore_amrwb.cpp
@@ -19,8 +19,8 @@
 #include "ADM_audioCodecEnum.h"
 
 extern "C" {
-#include <opencore-amrwb/dec_if.h>
-#include <opencore-amrwb/if_rom.h>
+#include <amrwb/dec_if.h>
+#include <amrwb/if_rom.h>
 };
 
 #define ADM_AMR_BUFFER (16 * 1024) // 16 kB internal
diff --git a/plugins/ADM_audioDecoders/ADM_ad_opencore_amrwb/ADM_ad_opencore_amrwb.cpp b/plugins/ADM_audioDecoders/ADM_ad_opencore_amrwb/ADM_ad_opencore_amrwb.cpp
index d13c25c..c98e7e1 100644
--- a/plugins/ADM_audioDecoders/ADM_ad_opencore_amrwb/ADM_ad_opencore_amrwb.cpp
+++ b/plugins/ADM_audioDecoders/ADM_ad_opencore_amrwb/ADM_ad_opencore_amrwb.cpp
@@ -105,7 +105,7 @@ again:
 			return 1;
 		}
 
-		D_IF_decode(state, buf, decodeBuffer, _good_frame);
+		D_IF_decode(state, (UWord8*) buf, decodeBuffer, _good_frame);
 
 		// int to float
 		int16_t *in = decodeBuffer;
