--- qt-everywhere-opensource-src-4.7.9999/src/gui/egl/qegl_p.h.orig	2010-04-18 02:48:47.000000000 +0300
+++ qt-everywhere-opensource-src-4.7.9999/src/gui/egl/qegl_p.h	2010-04-18 02:49:21.000000000 +0300
@@ -59,11 +59,8 @@
 #   include <GLES2/gl2.h>
 #endif
 
-#if defined(QT_GLES_EGL)
-#   include <GLES/egl.h>
-#else
-#   include <EGL/egl.h>
-#endif
+#include <EGL/egl.h>
+#include <EGL/eglext.h>
 
 #if defined(Q_WS_X11)
 // If <EGL/egl.h> included <X11/Xlib.h>, then the global namespace
