Index: mountmanager-0.2.6/configure
===================================================================
--- mountmanager-0.2.6.orig/configure
+++ mountmanager-0.2.6/configure
@@ -3,7 +3,7 @@
 # Copyright(C) 2007-2008 Tikhonov Sergey
 # configure script
 
-PACKAGE_NAME="MountManager"
+PACKAGE_NAME="mountmanager"
 PACKAGE_VERSION="0.2.6"
 
 echo "!---------------------------------------------------------!"
@@ -17,6 +17,7 @@ datadir="${prefix}/share"
 destdir="${datadir}/${PACKAGE_NAME}"
 include_path="${prefix}/include"
 lib_path="${prefix}/lib"
+fake_root="portage_d"
 qt4_test_dir="Qt4Test/"
 
 usage() {
@@ -374,18 +375,18 @@ OnlyShowIn=KDE;
 ############################################################
 #	Makefile
 ############################################################
-echo "FAKE_ROOT=
-BIN="${prefix}"/bin
-SHAREDIR="${prefix}"/share/mountmanager
+echo "FAKE_ROOT="${fake_root}"
+BIN="${fake_root}/${prefix}/bin"
+SHAREDIR="${fake_root}/${prefix}/share/mountmanager"
-OPTIONS=\$(SHAREDIR)/options
-ICONS=\$(SHAREDIR)/icons
-IMAGES=\$(SHAREDIR)/images
-DOC="${prefix}"/share/doc/MountManager
-APPLICATIONS="${prefix}"/share/applications
-MANS="${prefix}"/share/man/man1
-TRANSDIR="${prefix}"/lib/mountmanager/trans
-ALL_ICONS="${prefix}"/share/icons
-PLUGINS_DIR="${prefix}"/lib/mountmanager/plugins
+OPTIONS="${SHAREDIR}/options"
+ICONS="${SHAREDIR}/icons"
+IMAGES="${SHAREDIR}/images"
+DOC="${fake_root}/${prefix}/share/doc/MountManager"
+APPLICATIONS="${fake_root}/${prefix}/share/applications"
+MANS="${fake_root}/${prefix}/share/man/man1"
+TRANSDIR="${fake_root}/${prefix}/lib/mountmanager/trans"
+ALL_ICONS="${fake_root}/${prefix}/share/icons"
+PLUGINS_DIR="${fake_root}/${prefix}/lib/mountmanager/plugins"
 
 CHMOD="${chmod_cmd}"
 QMAKE="${qmake_cmd}"
