--- src/configure.orig	2008-11-08 01:07:52.000000000 +0100
+++ src/configure	2008-11-08 01:41:02.000000000 +0100
@@ -3,7 +3,7 @@
 # Copyright(C) 2007-2008 Tikhonov Sergey
 # configure script

-PACKAGE_NAME="MountManager"
+PACKAGE_NAME="mountmanager"
 PACKAGE_VERSION="0.2.5"

 echo "!---------------------------------------------------------!"
@@ -17,6 +17,7 @@
 destdir="${datadir}/${PACKAGE_NAME}"
 include_path="${prefix}/include"
 lib_path="${prefix}/lib"
+fake_root="portage_d"

 usage() {
 	echo ""
@@ -352,18 +353,18 @@
 ############################################################
 #	Makefile
 ############################################################
-echo "FAKE_ROOT=
-BIN="${prefix}"/bin
-SHAREDIR="${prefix}"/share/mountmanager
+echo "FAKE_ROOT="${fake_root}"
+BIN="${fake_root}/${prefix}"/bin
+SHAREDIR="${fake_root}/${prefix}"/share/mountmanager
 OPTIONS=\$(SHAREDIR)/options
 ICONS=\$(SHAREDIR)/icons
 IMAGES=\$(SHAREDIR)/images
-DOC="${prefix}"/share/doc/MountManager
-APPLICATIONS="${prefix}"/share/applications
-MANS="${prefix}"/share/man/man1
-TRANSDIR="${prefix}"/lib/mountmanager/trans
-ALL_ICONS="${prefix}"/share/icons
-PLUGINS_DIR="${prefix}"/lib/mountmanager/plugins
+DOC="${fake_root}/${prefix}"/share/doc/mountmanager-${PACKAGE_VERSION}
+APPLICATIONS="${fake_root}/${prefix}"/share/applications
+MANS="${fake_root}/${prefix}"/share/man/man1
+TRANSDIR="${fake_root}/${lib_path}"/mountmanager/trans
+ALL_ICONS="${fake_root}/${prefix}"/share/icons
+PLUGINS_DIR="${fake_root}/${lib_path}"/mountmanager/plugins

 CHMOD="${chmod_cmd}"
 QMAKE="${qmake_cmd}"
