--- src/playlistmodel.cpp	2009-01-21 09:22:49.000000000 +0200
+++ src/playlistmodel.cpp	2009-03-03 12:01:37.144486613 +0200
@@ -134,7 +134,7 @@
 QVariant PlaylistModel::data(int role) const
 {
 	if(role == Qt::BackgroundRole) {
-		return new QBrush(QColor::fromRgb(128, 200, 200));
+		return QVariant(QBrush(QColor::fromRgb(128, 200, 200)));
 	}
 	return QVariant();
 }
@@ -156,8 +156,8 @@
     }
     
     if(index.row() == _current) {
-    	if(role == Qt::FontRole) return new QFont("arial", -1, -1, true);
-    	else if(role == Qt::BackgroundRole) return new QBrush(QColor::fromRgb(128, 200, 200));
+    	if(role == Qt::FontRole) return QVariant(QFont("arial", -1, -1, true));
+    	else if(role == Qt::BackgroundRole) return QVariant(QBrush(QColor::fromRgb(128, 200, 200)));
     }
     
     switch(role) {
@@ -173,7 +173,7 @@
                                      int role) const
 {
 	if (role == Qt::BackgroundRole) {
-		return new QBrush(QColor::fromRgb(128, 200, 200));
+		return QVariant(QBrush(QColor::fromRgb(128, 200, 200)));
 	}
     if (role != Qt::DisplayRole)
         return QVariant();
