--- Cuberok.pro	2009-11-13 21:06:26.343766134 +0200
+++ Cuberok.pro1	2009-11-13 21:08:07.410636222 +0200
@@ -10,11 +10,6 @@
     CONFIG += disable_audiere disable_gst disable_phonon disable_ffmpeg
 }
 
-eval(QT_VERSION >= "4.5.0") {
-#    message($$[QT_VERSION])
-    CONFIG += player_phonon
-}
-
 player_phonon {
     disable_phonon {
         message(phonon disabled)
@@ -42,30 +37,26 @@
 }
 unix {
     CONFIG += link_pkgconfig
-    system(pkg-config --modversion libavcodec 2>/dev/null) {
-        system(pkg-config --modversion sdl 2>/dev/null) {
-            disable_ffmpeg {
-                message(ffmpeg disabled)
-            } else {
-                message(using ffmpeg backend)
-                SUBDIRS += plugins/player_ffmpeg
+    player_ffmpeg {
+        system(pkg-config --modversion libavcodec 2>/dev/null) {
+            system(pkg-config --modversion sdl 2>/dev/null) {
+                disable_ffmpeg {
+                    message(ffmpeg disabled)
+                } else {
+                    message(using ffmpeg backend)
+                    SUBDIRS += plugins/player_ffmpeg
+                }
             }
         }
     }
-    system(audiere-config --version) {
-        disable_audiere {
-            message(audiere disabled)
-        } else {
-            message(using audiere backend)
-            SUBDIRS += plugins/player_audiere
-        }
-    }
-    system(pkg-config --modversion gstreamer-0.10 2>/dev/null) {
-        disable_gst {
-            message(gst disabled)
-        } else {
-            message(using gstreamer backend)
-            SUBDIRS += plugins/player_gst
+    player_gstreamer {
+        system(pkg-config --modversion gstreamer-0.10 2>/dev/null) {
+            disable_gst {
+                message(gst disabled)
+            } else {
+                message(using gstreamer backend)
+                SUBDIRS += plugins/player_gst
+            }
         }
     }
 }
