--- Python/configure.py.orig	2009-05-13 23:54:57.000000000 +0200
+++ Python/configure.py	2009-05-13 23:56:26.000000000 +0200
@@ -276,7 +276,8 @@
             installs=installs,
             static=opts.static,
             debug=opts.debug,
-            universal=pyqt.universal
+            universal=pyqt.universal,
+            strip=0
         )
     else:
         makefile = Makefile(
@@ -285,7 +286,8 @@
             install_dir=opts.qscimoddir,
             installs=installs,
             static=opts.static,
-            debug=opts.debug
+            debug=opts.debug,
+            strip=0
         )

     if qsci_define:
