From f1b587d2d0f77ed8ea95bcf28880d231f048e0ee Mon Sep 17 00:00:00 2001
From: Jimi Huotari <chiitoo@gentoo.org>
Date: Wed, 5 Dec 2018 21:16:13 +0200
Subject: [PATCH] build: Make PolkitQt5-1 optional

---
 CMakeLists.txt | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 56d16c6..dfe90fe 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -41,8 +41,14 @@ find_package(lxqt-build-tools ${LXQTBT_MINIMUM_VERSION} REQUIRED)
 find_package(Qt5 ${QT_MINIMUM_VERSION} CONFIG REQUIRED Widgets DBus X11Extras LinguistTools)
 find_package(Qt5Xdg ${QTXDG_MINIMUM_VERSION} REQUIRED)
 find_package(KF5WindowSystem ${KF5_MINIMUM_VERSION} REQUIRED)
-find_package(PolkitQt5-1 REQUIRED)
 find_package(X11 REQUIRED)
+
+# Optionally include the PolkitQt5-1 module.
+option(BUILD_POLKIT "Install the PolkitQt5-1 files." ON)
+if(BUILD_POLKIT)
+    find_package(PolkitQt5-1 REQUIRED)
+endif ()
+
 message(STATUS "Building ${PROJECT_NAME} with Qt ${Qt5Core_VERSION}")
 
 include(CMakePackageConfigHelpers)
@@ -363,7 +369,9 @@ install(FILES ${LXQT_CONFIG_FILES}
     COMPONENT Runtime
 )
 
+if (PolkitQt5-1_FOUND)
 install(FILES ${POLKIT_FILES} DESTINATION "${POLKITQT-1_POLICY_FILES_INSTALL_DIR}")
+endif ()
 
 #************************************************
 # Create and install pkgconfig file
-- 
2.19.2

