Index: qt-everywhere-opensource-src-4.7.9999/configure
===================================================================
--- qt-everywhere-opensource-src-4.7.9999.orig/configure
+++ qt-everywhere-opensource-src-4.7.9999/configure
@@ -5392,15 +5392,6 @@ if [ "$PLATFORM_X11" = "yes" ]; then
 	QMakeVar add QMAKE_CXXFLAGS -fpermissive
         X11TESTS_FLAGS="$X11TESTS_FLAGS -fpermissive"
     fi
-
-    # Check we actually have X11 :-)
-    "$unixtests/compile.test" "$XQMAKESPEC" "$QMAKE_CONFIG" $OPT_VERBOSE "$relpath" "$outpath" config.tests/x11/xlib "XLib" $L_FLAGS $I_FLAGS $l_FLAGS $X11TESTS_FLAGS
-    if [ $? != "0" ]; then
-        echo "Basic XLib functionality test failed!"
-        echo " You might need to modify the include and library search paths by editing"
-        echo " QMAKE_INCDIR_X11 and QMAKE_LIBDIR_X11 in ${XQMAKESPEC}."
-        exit 1
-    fi
 fi
 
 # X11/MINGW OpenGL
