--- a/config.tests/xml2/xml2.cpp
+++ b/config.tests/xml2/xml2.cpp
@@ -27,9 +27,6 @@
 ****************************************************************************/
 
 #include <libxml/xmlversion.h>
-#if defined(LIBXML_CATALOG_ENABLED)
-#error "libxml catalog enabled"
-#endif
 #if !defined(LIBXML_ICU_ENABLED)
 #error "libxml icu not enabled"
 #endif
--- a/src/3rdparty/chromium/third_party/blink/renderer/core/xml/parser/xml_document_parser.cc
+++ b/src/3rdparty/chromium/third_party/blink/renderer/core/xml/parser/xml_document_parser.cc
@@ -28,6 +28,10 @@
 
 #include <libxml/parser.h>
 #include <libxml/parserInternals.h>
+#include <libxml/xmlversion.h>
+#if defined(LIBXML_CATALOG_ENABLED)
+#include <libxml/catalog.h>
+#endif
 #include <libxslt/xslt.h>
 
 #include <memory>
@@ -646,6 +650,9 @@ static void InitializeLibXMLIfNecessary() {
   if (did_init)
     return;
 
+#if defined(LIBXML_CATALOG_ENABLED)
+  xmlCatalogSetDefaults(XML_CATA_ALLOW_NONE);
+#endif
   xmlInitParser();
   xmlRegisterInputCallbacks(MatchFunc, OpenFunc, ReadFunc, CloseFunc);
   xmlRegisterOutputCallbacks(MatchFunc, OpenFunc, WriteFunc, CloseFunc);
