From https://codereview.qt-project.org/#/c/251995/

--- a/src/3rdparty/chromium/url/url_idna_icu.cc
+++ b/src/3rdparty/chromium/url/url_idna_icu.cc
@@ -86,8 +86,10 @@ bool IDNToASCII(const base::char16* src, int src_len, CanonOutputW* output) {
   while (true) {
     UErrorCode err = U_ZERO_ERROR;
     UIDNAInfo info = UIDNA_INFO_INITIALIZER;
-    int output_length = uidna_nameToASCII(uidna, src, src_len, output->data(),
-                                          output->capacity(), &info, &err);
+    int output_length = uidna_nameToASCII(uidna,
+        reinterpret_cast<const UChar*>(src), src_len,
+        reinterpret_cast<UChar*>(output->data()), output->capacity(),
+        &info, &err);
     if (U_SUCCESS(err) && info.errors == 0) {
       output->set_length(output_length);
       return true;
