Signed-off-by: Nicholas Meyer <nickaristocrates@gmail.com>

--- a/src/3rdparty/chromium/content/public/common/content_features.cc	2019-10-10 22:59:16.733542336 -0700
+++ b/src/3rdparty/chromium/content/public/common/content_features.cc	2019-10-10 22:59:03.675634565 -0700
@@ -119,12 +119,7 @@
 // Enables crash reporting via Reporting API.
 // https://www.w3.org/TR/reporting/#crash-report
 const base::Feature kCrashReporting{"CrashReporting",
-#if BUILDFLAG(ENABLE_REPORTING)
-                                    base::FEATURE_ENABLED_BY_DEFAULT
-#else
-                                    base::FEATURE_DISABLED_BY_DEFAULT
-#endif
-};
+                                    base::FEATURE_DISABLED_BY_DEFAULT};
 
 // Enables specification of a target element in the fragment identifier
 // via a CSS selector.
