diff -Naur qtjambi-src-preview-4.5.0_01~orig/build.xml qtjambi-src-preview-4.5.0_01/build.xml
--- qtjambi-src-preview-4.5.0_01~orig/build.xml	2009-05-01 14:37:12.000000000 +0200
+++ qtjambi-src-preview-4.5.0_01/build.xml	2009-05-01 14:40:17.000000000 +0200
@@ -293,8 +293,7 @@
     </target>
 
     <target name="library.designer.bundle" 
-	    depends="library.designer.compile" 
-	    unless="qtjambi.library.cppfiles.uptodate">
+	    depends="library.designer.compile">
         <jar destfile="${outputDir}/qtjambi-designer-${qtjambi.version}.jar"
              basedir="${outputDir}">
             <include name="com/trolltech/tools/designer/*"/>
@@ -593,7 +592,9 @@
     <target name="init"
         depends="init.ant, init.taskdef"
         description="Initializes the Qt Jambi build environment." >
-      <qtjambi-initialize verbose="true" configuration="${qtjambi.config}" />
+      <qtjambi-initialize verbose="true" configuration="${qtjambi.config}"
+			opengl="${gentoo.opengl}" phonon="${gentoo.phonon}"
+			webkit="${gentoo.webkit}" xmlpatterns="${gentoo.xmlpatterns}"/>
     </target>
 
     <target name="generator"
diff -Naur qtjambi-src-preview-4.5.0_01~orig/com/trolltech/tools/ant/InitializeTask.java qtjambi-src-preview-4.5.0_01/com/trolltech/tools/ant/InitializeTask.java
--- qtjambi-src-preview-4.5.0_01~orig/com/trolltech/tools/ant/InitializeTask.java	2009-05-01 14:37:11.000000000 +0200
+++ qtjambi-src-preview-4.5.0_01/com/trolltech/tools/ant/InitializeTask.java	2009-05-01 14:44:57.000000000 +0200
@@ -105,6 +105,22 @@
         return configuration;
     }
 
+    public void setOpengl(String opengl) {
+	this.opengl = opengl;
+    }
+
+    public void setPhonon(String phonon) {
+	this.phonon = phonon;
+    }
+
+    public void setWebkit(String webkit) {
+	this.webkit = webkit;
+    }
+
+    public void setXmlpatterns(String xmlpatterns) {
+	this.xmlpatterns = xmlpatterns;
+    }
+
     public void execute() throws BuildException {
         props = PropertyHelper.getPropertyHelper(getProject());
         props.setNewProperty(null, OSNAME, decideOSName());
@@ -301,7 +317,7 @@
     }
 
     private String decideLibSubDir() {
-        libSubDir = OSInfo.os() == OSInfo.OS.Windows ? "bin" : "lib";
+        libSubDir = "";
         if (verbose) System.out.println("qtjambi.libsubdir: " + libSubDir);
         return libSubDir;
     }
@@ -330,8 +346,6 @@
         StringBuilder path = new StringBuilder();
         path.append(props.getProperty(null, QTDIR));
         path.append("/");
-        path.append(props.getProperty(null, LIBSUBDIR));
-        path.append("/");
         path.append(LibraryEntry.formatQtName(name, debug, version));
         return new File(path.toString()).exists();
     }
@@ -353,25 +367,25 @@
     }
 
     private String decidePhonon() {
-        String result = String.valueOf(doesQtLibExist("phonon", 4));
+        String result = phonon;
         if (verbose) System.out.println(PHONON + ": " + result);
         return result;
     }
 
     private String decideWebkit() {
-        String result = String.valueOf(doesQtLibExist("QtWebKit", 4));
+        String result = webkit;
         if (verbose) System.out.println(WEBKIT + ": " + result);
         return result;
     }
 
     private String decideXMLPatterns() {
-        String result = String.valueOf(doesQtLibExist("QtXmlPatterns", 4));
+        String result = xmlpatterns;
         if (verbose) System.out.println(XMLPATTERNS + ": " + result);
         return result;
     }
 
     private String decideOpenGL() {
-        String result = String.valueOf(doesQtLibExist("QtOpenGL", 4));
+        String result = opengl;
         if (verbose) System.out.println(OPENGL + ": " + result);
         return result;
     }
@@ -385,4 +399,8 @@
     private String qmakespec;
     private String qtdir;
     private String libSubDir;
+    private String opengl;
+    private String phonon;
+    private String webkit;
+    private String xmlpatterns;
 }
