--- qtjambi-src-lgpl-4.5.0_01.orig/juic/javautils.cpp	2009-07-19 19:17:42.000000000 +0200
+++ qtjambi-src-lgpl-4.5.0_01/juic/javautils.cpp	2009-07-19 19:10:17.000000000 +0200
@@ -43,6 +43,8 @@
 #include "javautils.h"
 #include <QtCore/QHash>
 
+#include <cstdio>
+
 QString javaFixString(const QString &str)
 {
     const ushort *chars = str.utf16();
--- qtjambi-src-lgpl-4.5.0_01.orig/generator/parser/rpp/pp-internal.h	2009-07-19 19:17:42.000000000 +0200
+++ qtjambi-src-lgpl-4.5.0_01/generator/parser/rpp/pp-internal.h	2009-07-19 19:10:17.000000000 +0200
@@ -46,6 +46,8 @@
 
 #include <algorithm>
 
+#include <cstdio>
+
 namespace rpp {
 
 namespace _PP_internal
--- qtjambi-src-lgpl-4.5.0_01.orig/qtjambi/qtjambitypemanager.cpp	2009-07-19 19:17:42.000000000 +0200
+++ qtjambi-src-lgpl-4.5.0_01/qtjambi/qtjambitypemanager.cpp	2009-07-19 19:10:18.000000000 +0200
@@ -637,8 +637,8 @@
     jvalue returned;
     returned.l = 0;
 
-    char *methodName;
-    char *signature;
+    const char *methodName;
+    const char *signature;
     switch (typeId) {
     case Integer:
         methodName = "intValue";
