diff -Naur qtjambi-src-preview-4.5.0_01.orig/generator/main.h qtjambi-src-preview-4.5.0_01/generator/main.h
--- qtjambi-src-preview-4.5.0_01.orig/generator/main.h	2009-02-10 12:44:47.000000000 +0100
+++ qtjambi-src-preview-4.5.0_01/generator/main.h	2009-02-25 22:57:37.000000000 +0100
@@ -68,18 +68,13 @@
             includes += commandLineIncludes.split(path_splitter);
 
         // Include Qt
-        QString qtdir = getenv ("QTDIR");
-        if (qtdir.isEmpty()) {
-            qWarning("QTDIR environment variable not set. This may cause problems with finding the necessary include files.");
-        } else {
-            qtdir += "/include";
-            includes << (qtdir + "/QtXml");
-            includes << (qtdir + "/QtNetwork");
-            includes << (qtdir + "/QtCore");
-            includes << (qtdir + "/QtGui");
-            includes << (qtdir + "/QtOpenGL");
-            includes << qtdir;
-        }
+        QString qtdir = "/usr/include/qt4";
+        includes << (qtdir + "/QtXml");
+        includes << (qtdir + "/QtNetwork");
+        includes << (qtdir + "/QtCore");
+        includes << (qtdir + "/QtGui");
+        includes << (qtdir + "/QtOpenGL");
+        includes << qtdir;
 
         foreach (QString include, includes)
             preprocess.push_include_path(QDir::convertSeparators(include).toStdString());
