diff -ru ../old/modules/mediaplayer/CMakeLists.txt ./modules/mediaplayer/CMakeLists.txt
--- ../old/modules/mediaplayer/CMakeLists.txt	2009-08-07 13:41:28.000000000 +0200
+++ ./modules/mediaplayer/CMakeLists.txt	2009-09-15 03:03:51.000000000 +0200
@@ -62,6 +62,7 @@
 add_custom_target (mediaplayer-translations DEPENDS ${TRANSLATION_FILES})
 set_target_properties(mediaplayer PROPERTIES COMPILE_FLAGS "-DMEDIAPLAYER_BUILD" )
 add_dependencies (mediaplayer mediaplayer-translations)
+target_link_libraries (mediaplayer ${QT_QTDBUS_LIBRARY})
 if (mediaplayer STREQUAL STATIC)
 	set_target_properties(mediaplayer PROPERTIES COMPILE_FLAGS "-DKADULIB" )
 else (mediaplayer STREQUAL STATIC)
