--- menu-cache/docs/reference/libmenu-cache/Makefile.am 2014-03-07 16:19:48.801475326 -0500
+++ menu-cache-9999/docs/reference/libmenu-cache/Makefile.am    2014-03-07 16:18:23.143473032 -0500
@@ -82,11 +82,12 @@
 GTKDOC_LIBS=$(top_builddir)/libmenu-cache/libmenu-cache.la

 # This includes the standard gtk-doc make rules, copied by gtkdocize.
-include $(top_srcdir)/gtk-doc.make
-
+if ENABLE_GTK_DOC
+       include $(top_srcdir)/gtk-doc.make
+endif
 # Other files to distribute
 # e.g. EXTRA_DIST += version.xml.in
-EXTRA_DIST +=
+#EXTRA_DIST =

 # Files not to distribute
 # for --rebuild-types in $(SCAN_OPTIONS), e.g. $(DOC_MODULE).types
