Index: qt-creator-1.0.0-src/src/plugins/plugins.pro
===================================================================
--- qt-creator-1.0.0-src.orig/src/plugins/plugins.pro
+++ qt-creator-1.0.0-src/src/plugins/plugins.pro
@@ -26,7 +26,9 @@ SUBDIRS   = plugin_coreplugin \
             plugin_qtscripteditor \
 #            plugin_cpaster \
             plugin_cmakeprojectmanager \
-            plugin_fakevim
+            plugin_fakevim \
+	    plugin_designer \
+	    plugin_resourceeditor
 
 # These two plugins require private headers from Qt and therefore don't work
 # with an installed/released version of Qt.
