diff --git a/doc/doc.pri b/doc/doc.pri
index 0deac0f..4cd350b 100644
--- a/doc/doc.pri
+++ b/doc/doc.pri
@@ -1,4 +1,4 @@
-unix:QDOC_BIN = $$(QTDIR)/bin/qdoc3
+unix:QDOC_BIN = /usr/bin/qdoc3
 win32:QDOC_BIN = $$(QTDIR)/bin/qdoc3.exe
 win32:QDOC_BIN = $$replace(QDOC_BIN, "/", "\\")
 
@@ -17,7 +17,7 @@ win32:QDOC_BIN = $$replace(QDOC_BIN, "/", "\\")
 
 unix {
     QDOC = SRCDIR=$$PWD OUTDIR=$$OUT_PWD/doc/html $$QDOC_BIN 
-    HELPGENERATOR = $$(QTDIR)/bin/qhelpgenerator
+    HELPGENERATOR = /usr/bin/qhelpgenerator
 } else {
     QDOC = set SRCDIR=$$PWD&& set OUTDIR=$$OUT_PWD/doc/html&& $$QDOC_BIN 
     HELPGENERATOR = $$(QTDIR)\bin\qhelpgenerator.exe
