--- qt-creator-0.9.1_beta.orig/src/plugins/help/helpplugin.cpp
+++ qt-creator-0.9.1_beta/src/plugins/help/helpplugin.cpp
@@ -439,12 +439,7 @@ void HelpPlugin::extensionsInitialized()
         QFileInfo fi(m_helpEngine->collectionFile());
 
         const QString qchFileName =
-            QDir::cleanPath(QCoreApplication::applicationDirPath()
-#if defined(Q_OS_MAC)
-            + QLatin1String("/../Resources/doc/qtcreator.qch"));
-#else
-            + QLatin1String("/../doc/qtcreator.qch"));
-#endif
+	QDir::cleanPath(QLatin1String("/usr/share/docs/qt-creator/qtcreator.qch"));
         QHelpEngineCore hc(fi.absoluteFilePath());
         hc.setupData();
         if (!hc.registerDocumentation(qchFileName))
