diff -up qt-creator-2.1.0-rc1-src/src/plugins/qmldesigner/qtquickplugin/plugindestdir.pri.qtquickplugin-lib-fix qt-creator-2.1.0-rc1-src/src/plugins/qmldesigner/qtquickplugin/plugindestdir.pri
--- qt-creator-2.1.0-rc1-src/src/plugins/qmldesigner/qtquickplugin/plugindestdir.pri.qtquickplugin-lib-fix	2010-11-24 17:28:38.000000000 +0200
+++ qt-creator-2.1.0-rc1-src/src/plugins/qmldesigner/qtquickplugin/plugindestdir.pri	2010-11-30 14:13:19.065191548 +0200
@@ -1,5 +1,7 @@
 macx {
   DESTDIR = $$IDE_LIBRARY_PATH/QmlDesigner
 } else {
-  DESTDIR = $$IDE_BUILD_TREE/lib/qmldesigner
+  DESTDIR = $$IDE_BUILD_TREE/$$IDE_LIBRARY_BASENAME/qmldesigner
+  target.path = /$$IDE_LIBRARY_BASENAME/qmldesigner
+  INSTALLS += target
 }
