Index: qt-creator-0.9.1_beta/qtcreator.pro
===================================================================
--- qt-creator-0.9.1_beta.orig/qtcreator.pro
+++ qt-creator-0.9.1_beta/qtcreator.pro
@@ -2,3 +2,12 @@ TEMPLATE  = subdirs
 CONFIG   += ordered
 
 SUBDIRS = src
+
+bin.files += bin/qtcreator
+bin.files += bin/runInTerminal.command
+libs.files += lib/*
+docs.files += doc/*
+
+INSTALLS += bin
+INSTALLS += libs
+INSTALLS += docs
