--- CMakeLists.txt
+++ CMakeLists.txt
@@ -2,7 +2,7 @@
 PROJECT( kajammer )
 
 SET( CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMake )
-SET(CMAKE_CXX_FLAGS "-g -Wall")
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
 
 FIND_PACKAGE( Qt4 REQUIRED )
 FIND_PACKAGE( Phonon REQUIRED )
@@ -90,6 +90,6 @@
 )
 
 INSTALL(TARGETS kajammer DESTINATION /usr/bin)
-INSTALL(FILES res/kajammer.png DESTINATION /usr/share/icons)
+INSTALL(FILES res/kajammer.png DESTINATION /usr/share/pixmaps)
 INSTALL(FILES res/kajammer.desktop DESTINATION /usr/share/applications)
 
