diff --git a/pylupdate/CMakeLists.txt b/pylupdate/CMakeLists.txt
index 7d09b98..0b34e80 100644
--- a/pylupdate/CMakeLists.txt
+++ b/pylupdate/CMakeLists.txt
@@ -20,7 +20,8 @@ include_directories(pyside-lupdate
                     ${QT_INCLUDE_DIR}
                     ${QT_QT_INCLUDE_DIR}
                     ${QT_QTXML_INCLUDE_DIR}
-                    ${QT_QTCORE_INCLUDE_DIR})
+                    ${QT_QTCORE_INCLUDE_DIR}
+                    ${QT_QTGUI_INCLUDE_DIR})
 
 target_link_libraries(pyside-lupdate
                       ${QT_QTCORE_LIBRARY}
