diff --git a/demos/demos.pro b/demos/demos.pro
index e1cb827..efb0a6e 100644
--- a/demos/demos.pro
+++ b/demos/demos.pro
@@ -12,16 +12,19 @@ SUBDIRS     =  \
     animatedlayout/translations \
     qtstyleexample \
     qtstyledialogs \
-    qtstylekinetic \
-    appletinstallationsource \
-    applicationextension \
-    applicationextension/demoextension \
-    applicationextension/demoextension2    
+    qtstylekinetic
 }
 
 # we do not have compiled QtDBusd4
-macx:SUBDIRS -= appletinstallationsource applicationextension applicationextension/demoextension applicationextension/demoextension2
-win32:SUBDIRS -= appletinstallationsource applicationextension applicationextension/demoextension applicationextension/demoextension2
+!macx:!win32 {
+    contains(DEFINES, HAVE_DBUS) {
+        SUBDIRS += \
+        appletinstallationsource \
+        applicationextension \
+        applicationextension/demoextension \
+        applicationextension/demoextension2
+    }
+}
 
 QMAKE_EXTRA_TARGETS += check
 check.commands = $$system(true)
