diff --git a/src/src.pro b/src/src.pro
index 9b3ddd2..dbfeb6f 100644
--- a/src/src.pro
+++ b/src/src.pro
@@ -7,12 +7,3 @@ SUBDIRS += \
     extensions
 win32|macx:SUBDIRS -= settings extensions
 
-unix {
-  include(../mkspecs/common.pri)
-  schemas.CONFIG += no_check_exist
-  schemas.files = $${IN_PWD}/meegotouch.schemas
-  schemas.path = $${M_SYSCONFDIR}/gconf/schemas
-  schemas.commands = GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) gconftool-2 --makefile-install-rule $${IN_PWD}/meegotouch.schemas
-
-  INSTALLS += schemas
-}
