diff --git a/src/src.pro b/src/src.pro
index f60a9dc..5cea0c7 100644
--- a/src/src.pro
+++ b/src/src.pro
@@ -9,17 +9,6 @@ win32|macx:SUBDIRS -= settings extensions
 
 include(../mkspecs/common.pri)
 
-contains(DEFINES, HAVE_GCONF) {
-    # install gconf schema
-    unix {
-        schemas.CONFIG += no_check_exist
-        schemas.files = $${IN_PWD}/data/meegotouch.schemas
-        schemas.path = $${M_SYSCONFDIR}/gconf/schemas
-        schemas.commands = GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) gconftool-2 --makefile-install-rule $${IN_PWD}/data/meegotouch.schemas
-        INSTALLS += schemas
-    }
-}
-
 #install device configuration file
 devices.CONFIG += no_check_exist
 devices.files = $${IN_PWD}/data/devices.conf
