diff --git a/configure b/configure
index 594e4eb..72a9031 100755
--- a/configure
+++ b/configure
@@ -364,50 +364,6 @@ if [ "$CFG_ICU" != "no" ]; then
     fi
 fi
 
-# Test for pkg-config enabled dependencies
-which pkg-config > /dev/null
-if [ $? -eq 0 ]; then
-
-    pkg-config --exists contextsubscriber-1.0
-    if [ $? -eq 0 ]; then
-        HAVE_CONTEXTSUBSCRIBER=yes
-    fi
-
-    if [ "$CFG_GCONF" != "no" ]; then
-        pkg-config --exists gconf-2.0
-        if [ $? -eq 0 ]; then
-            HAVE_GCONF=yes
-        else
-            if [ "$CFG_GCONF" = "yes" ]; then
-                echo "You specified  -gconf  but gconf was not found."
-                echo "Please install the libgconf2-dev package."
-                exit 1
-            fi
-        fi
-    fi
-
-    pkg-config --exists gstreamer-0.10
-    if [ $? -eq 0 ]; then
-	HAVE_GSTREAMER=yes
-    fi
-
-    pkg-config --exists QtDBus
-    if [ $? -eq 0 ]; then
-	HAVE_DBUS=yes
-    fi    
-
-    pkg-config --exists contentaction-0.1
-    if [ $? -eq 0 ]; then
-	#HAVE_CONTENTACTION=yes DISABLED FOR NOW
-	HAVE_CONTENTACTION=no
-    fi
-
-    if [ "$SBOX_UNAME_MACHINE" = "arm" ]; then
-        IS_ARMEL=yes
-    fi
-
-fi
-
 
 
 #-------------------------------------------------------------------------------
