From b39c4a03503aa1449efaae0c827fd207d8c96992 Mon Sep 17 00:00:00 2001
From: Michael <kensington@astralcloak.net>
Date: Thu, 5 Apr 2012 21:58:03 +1000
Subject: [PATCH 1/2] Avoid collision with minor/major macro definitions in
 glibc.

---
 src/Utils/src/Version.cpp | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/src/Utils/src/Version.cpp b/src/Utils/src/Version.cpp
index a41116b..5f4c2a6 100644
--- a/src/Utils/src/Version.cpp
+++ b/src/Utils/src/Version.cpp
@@ -22,6 +22,14 @@
 #include <QtCore/QRegExp>
 #include "Version.h"
 
+#ifdef minor
+#undef minor
+#endif
+
+#ifdef major
+#undef major
+#endif
+
 class VersionPrivate
 {
 public:
-- 
1.7.12

