From 5320cf9b5c764d621b7135a13d84d1cd9f63e2df Mon Sep 17 00:00:00 2001
From: Michael <kensington@astralcloak.net>
Date: Thu, 5 Apr 2012 22:05:37 +1000
Subject: [PATCH 2/2] Fix screenie.desktop to pass validation.

---
 src/Screenie/res/screenie.desktop | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/src/Screenie/res/screenie.desktop b/src/Screenie/res/screenie.desktop
index aa316d3..d2f6988 100644
--- a/src/Screenie/res/screenie.desktop
+++ b/src/Screenie/res/screenie.desktop
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Version=0.1.0
+Version=1.0
 Type=Application
 Name=Screenie
 GenericName=Screenshot composer
@@ -8,7 +8,6 @@ Exec=Screenie %U
 TryExec=Screenie
 Icon=screenie
 Terminal=false
-Categories=Graphics;2DGraphics;RasterGraphics;Qt
+Categories=Graphics;2DGraphics;RasterGraphics;Qt;
 StartupNotify=true
-MimeType=application/x-xsc
-
+MimeType=application/x-xsc;
-- 
1.7.12

